packages/http/src/validator/validators/__tests__/__snapshots__/query.spec.ts.snap

```diff
@@ -1,28 +1,28 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`HttpQueryValidator validate() spec is present header is present deprecated flag is set returns deprecation warning 1`] = `
+exports[`HttpQueryValidator validate() spec is present query param is not present spec defines it as required returns validation error 1`] = `
 Array [
   Object {
-    "code": "deprecated",
-    "message": "Query param param is deprecated",
+    "code": "required",
+    "message": "should have required property 'aparam'",
     "path": Array [
       "query",
-      "param",
     ],
-    "severity": 1,
+    "severity": 0,
   },
 ]
 `;
 
-exports[`HttpQueryValidator validate() spec is present query param is not present spec defines it as required returns validation error 1`] = `
+exports[`HttpQueryValidator validate() spec is present query param is present deprecated flag is set returns deprecation warning 1`] = `
 Array [
   Object {
-    "code": "required",
-    "message": "should have required property 'aparam'",
+    "code": "deprecated",
+    "message": "Query param param is deprecated",
     "path": Array [
       "query",
+      "param",
     ],
-    "severity": 0,
+    "severity": 1,
```

Comment:
Aha ok it took me ages to understand what was going here — all it's happening is simply reversing the order :trollface:

Reply:
Haha, me too ;) I should have comment it somehow to save reviewers time.